experimental/PropertyDDS/packages/property-properties/src/index.d.ts

```diff
@@ -2128,7 +2134,7 @@ declare module "@fluid-experimental/property-properties" {
           /**
            * Inserts a property into the set
            */
-          insert(): void;
+          insert(in_property: NamedProperty): void;
```

Comment:
Should this be "| string" also?

Reply:
no from the JS api it only expect a NamedProperty , the set property uses intenral guids to uniquely identify the properties so a user would never specify a key.